Policies/github-terms-of-service.md

```diff
@@ -133,20 +134,22 @@ You may create or upload User-Generated Content while using the Service. You are
+Because GitHub does not User-Generated Content, We — and Other Users — need certain legal permissions, as listed in Sections D.4 — D.7. You understand that You will not receive any payment for any of the rights discussed in Sections D.4 — D.7. Any permissions You grant to Us and Other Users will end when You remove Your Content from our servers.
 
 #### 4. License Grant to Us
-We need the legal right to do things like host Your Content, publish it, and share it. You grant us and our legal successors the right to store, parse, and display Your Content, and make incidental copies as necessary to render the Website and provide the Service. This includes the right to do things like copy it to our database and make backups; show it to you and other users; parse it into a search index or otherwise analyze it on our servers; share it with other users; and perform it, in case Your Content is something like music or video.
+We need the legal right to store, parse, and display Your Content, and make incidental copies as necessary to render the Website and provide the Service. This includes the right to do things like copy it to our database and make backups; show it to You and Other Users; parse it into a search index or otherwise analyze it on our servers; share it with Other Users; and perform it, in case Your Content is something like music or video.
 
-This license does not grant GitHub the right to sell Your Content or otherwise distribute or use it outside of our provision of the Service.
+We do not need the right to sell Your Content or otherwise distribute or use it outside of our provision of the Service.
+
+If Your Content's existing license grants Us the needed permissions, no additional license is required. If Your Content's existing license does not grant Us the needed permissions, then you agree grant Us those permissions.
 
 #### 5. License Grant to Other Users
-Any User-Generated Content you post publicly, including issues, comments, and contributions to other Users' repositories, may be viewed by others. By setting your repositories to be viewed publicly, you agree to allow others to view and "fork" your repositories (this means that others may make their own copies of Content from your repositories in repositories they control).
+Other Users need the legal right to view Content they are authorized to access via the Website or Service. For example, all users need the legal right to access issues, comments, and contributions to public repositories.
```

Comment:
> define “access”, reading? modifying? creating new ones?

I don't think we need to define “acccess”.  It can contain all of the interpretations you suggest, but they only need the right to *view* that content.  There's a theoretical chance that GitHub would authorize a user to modify content that they cannot view, but I don't think you need to make that distinction for Your Content.  And if the Other User is creating new content, then they already have full access to it, so You don't need to grant them any new permissions.

> Why do they need that?

So they can view the site.

> Case in point, I can disable issues on my repositories and usually do, especially for those that are just mirrors.

If you disable issues before any are created, you are removing Other Users' right to post to that location.  That does not affect Other Users right to view anything.

If you disable issues after some have been created, you may be removing an Other User's right to view it.  That's the same as deleting an Other User's comment in your issue tracker.  There's some wording about removing content in [the current §D.3][1], but we may need a new section to cover User-initiated removal.  Once that is added, it will cover both disabling a populated issue tracker and deleting individual issues.  But that seems different enough that I'll punt on it for this PR.

[1]: https://github.com/github/site-policy/blob/75c016d80fe819c192f5c953eb3829bee63462a2/Policies/github-terms-of-service.md#3-github-may-remove-content